sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -9,5 +16,81 @@ public ConversationAnalysisClientLiveTests(bool isAsync, ConversationAnalysisCli
             : base(isAsync, serviceVersion, null /* RecordedTestMode.Record /* to record */)
         {
         }
+        private static string EnglishText = "We'll have 2 plates of seared salmon nigiri.";
+        private static string SpanishText = "Tendremos 2 platos de nigiri de salmón braseado.";
+        private static List<string> ExpectedOutput = new List<string>()
+        {
+            "2 plates",
+            "seared salmon nigiri"
+        };
+
+        [RecordedTest]
+        public async Task AnalyzeConversation()
+        {
+            AnalyzeConversationOptions options = new AnalyzeConversationOptions(
+               TestEnvironment.ProjectName,
+               TestEnvironment.DeploymentName,
+               EnglishText);
+
+            Response<AnalyzeConversationResult> response = await Client.AnalyzeConversationAsync(options);
+
+            Assert.That(response.Value.Prediction.TopIntent, Is.EqualTo("Order"));
+            Assert.That(response.Value.Prediction.ProjectKind, Is.EqualTo(ProjectKind.Conversation));
+        }
+
+        [RecordedTest]
+        public async Task AnalyzeConversationWithLanguage()
+        {
+            AnalyzeConversationOptions options = new AnalyzeConversationOptions(
+               TestEnvironment.ProjectName,
+               TestEnvironment.DeploymentName,
+               SpanishText)
+            {
+                Language = "es"
+            };
+
+            Response<AnalyzeConversationResult> response = await Client.AnalyzeConversationAsync(options);
+
+            Assert.That(response.Value.Prediction.TopIntent, Is.EqualTo("Order"));
+            Assert.That(response.Value.Prediction.ProjectKind, Is.EqualTo(ProjectKind.Conversation));
+        }
+
+        [RecordedTest]
+        public async Task AnalyzeConversationsDeepstack()
+        {
+            AnalyzeConversationOptions options = new AnalyzeConversationOptions(
+               TestEnvironment.ProjectName,
+               TestEnvironment.DeploymentName,
+               EnglishText);
+
+            Response<AnalyzeConversationResult> response = await Client.AnalyzeConversationAsync(options);
+
+            DeepstackPrediction deepstackPrediction = response.Value.Prediction as DeepstackPrediction;
+
+            Assert.That(response.Value.Prediction.ProjectKind, Is.EqualTo(ProjectKind.Conversation));
+
+            Assert.That(deepstackPrediction.TopIntent, Is.EqualTo("Order"));
+
+            IList<string> entitiesText = deepstackPrediction.Entities.Select(entity => entity.Text).ToList();
+            Assert.That(entitiesText, Has.Count.EqualTo(2));
+            Assert.That(entitiesText, Is.EquivalentTo(ExpectedOutput));
```

Comment:
I used `Is.EquivalentTo` to assert it. Works well.

Reply:
You actually don't need to assert the count. `EquivalentTo` does that implicitly. This is fine, though.